content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md

```diff
@@ -0,0 +1,312 @@
+          caBundle: <CABundle info>
+      conversionReviewVersions:
+      - v1
+      - v2
+  ```
+  Create CRD using kubectl
+  ```shell
+  kubectl apply -f test-crd.yaml
+  ```
+- Create a manifest for an example testcrd. Name the manifest `cr1.yaml` and use these contents:
+  ```yaml
+  apiVersion: stable.example.com/v1
+  kind: SelfieRequest
+  metadata:
+    name: cr1
+    namespace: default
+  ```
+  Create CR using kubectl
+  ```shell
+  kubectl apply -f cr1.yaml
+  ```
+- Verify that CR is written and stored as v1 by getting the object from etcd.
+  ```shell
+  ETCDCTL_API=3 etcdctl get /kubernetes.io/stable.example.com/testcrds/default/cr1 [...] | hexdump -C
+  ```
+  where [...] contains the additional arguments for connecting to the etcd server.
+- Update the CRD `test-crd.yaml` to include v2 version for serving and storage
+ and v1 as serving only, as follows:
+  ```yaml
+  apiVersion: apiextensions.k8s.io/v1
+  kind: CustomResourceDefinition
+  metadata:
+  name: selfierequests.stable.example.com
+  spec:
+    group: stable.example.com
+    names:
+      plural: SelfieRequests
+      singular: SelfieRequest
+      kind: SelfieRequest
+      listKind: SelfieRequestList
+    scope: Namespaced
+    versions:
+      - name: v2
+        served: true
+        storage: true
+        schema:
+          openAPIV3Schema:
+            type: object
+            properties:
+              host:
+                type: string
+              port:
+                type: string
+      - name: v1
+        served: true
+        storage: false
+        schema:
+          openAPIV3Schema:
+            type: object
+            properties:
+              hostPort:
+                type: string
+    conversion:
+      strategy: Webhook
+      webhook:
+        clientConfig:
+          url: 'https://127.0.0.1:9443/crdconvert'
+          caBundle: <CABundle info>
+        conversionReviewVersions:
+          - v1
+          - v2
+  ```
+  Update CRD using kubectl
+  ```shell
+  kubectl apply -f test-crd.yaml
+  ```
+- Create CR resource file with name `cr2.yaml` as follows:
+  ```yaml
+  apiVersion: stable.example.com/v2
+  kind: SelfieRequest
+  metadata:
+    name: cr2
+    namespace: default
+  ```
+- Create CR using kubectl
+  ```shell
+  kubectl apply -f cr2.yaml
+  ```
+- Verify that CR is written and stored as v2 by getting the object from etcd.
+  ```shell
+  ETCDCTL_API=3 etcdctl get /kubernetes.io/stable.example.com/testcrds/default/cr2 [...] | hexdump -C
+  ```
+  where [...] contains the additional arguments for connecting to the etcd server.
+- Create `StorageVersionMigration` resource file with the name `migrate-crd.yaml`
+as follows:
+  ```yaml
+  kind: StorageVersionMigration
+  apiVersion: storagemigration.k8s.io/v1alpha1
+  metadata:
+  name: crdsvm
+  spec:
+    resource:
+      group: stable.example.com
+      version: v1
+      resource: SelfieRequest
+  ```
+  Create the resource using _kubectl_ as follows:
```

Comment:
```suggestion
  Create the object using _kubectl_ as follows:
```
We know that this resource is also an object (it has `.metadata`, `.spec` and `.status`)